.github/workflows/build-lint-test.yml

```diff
@@ -132,8 +132,22 @@ jobs:
           key: test-build-${{ runner.os }}-${{ matrix.node-version }}-${{ github.sha }}
       - run: yarn --immutable --immutable-cache
       - run: yarn workspace ${{ matrix.package-name }} run test:ci
-      - name: Upload coverage results
-        uses: codecov/codecov-action@d9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70
+      - name: Get coverage folder
+        id: get-coverage-folder
+        run: |
+          echo "stub" >> stub
+          echo "coverage-folder=$(yarn workspaces list --json | grep ${{ matrix.package-name }} | jq -r '.location')/coverage" >> "$GITHUB_OUTPUT"
+        shell: bash
+      - name: Upload coverage artifact
+        if: ${{ matrix.node-version == '16.x' }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage
+          path: |
+            stub
+            ${{ steps.get-coverage-folder.outputs.coverage-folder }}/**/coverage-final.json
+          if-no-files-found: warn
```

Comment:
Should we throw an error here instead?

Reply:
The problem is that we run a job for `snaps-types`, which doesn't have any tests. If this was set to `error`, it would fail, as you can see here:

https://github.com/MetaMask/snaps-monorepo/actions/runs/4322427700/jobs/7544863506

Reply:
Ah, fair enough.